app/tasks/maintenance/move_image_urls_task.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Maintenance
+  class MoveImageUrlsTask < MaintenanceTasks::Task
+    def collection
+      Podcast.all
+    end
+
+    def process(podcast)
+      if podcast.extras["image_urls"].nil?
```

Comment:
return early `return if podcast.extras["image_urls"].present?`

Reply:
Good point. We don't need to update podcasts' extras if they already have.